src/locationTracker/mapTracker/EntranceMarker.tsx

```diff
@@ -0,0 +1,226 @@
+import { CSSProperties, MouseEvent, useCallback } from 'react';
+import Tippy from '@tippyjs/react';
+import { followCursor } from 'tippy.js';
+import 'react-contexify/dist/ReactContexify.css';
+import { useContextMenu } from 'react-contexify';
+import AreaCounters from '../AreaCounters';
+import Logic from '../../logic/Logic';
+import LogicHelper from '../../logic/LogicHelper';
+import ColorScheme from '../../customization/ColorScheme';
+import { MarkerClickCallback, HintClickCallback, EntranceBindCallback, CheckAllClickCallback } from '../../callbacks';
+import keyDownWrapper from '../../KeyDownWrapper';
+
+import sotsImage from '../../assets/hints/sots.png';
+import barrenImage from '../../assets/hints/barren.png';
+
+import g1 from '../../assets/hints/g1.png';
+import scaldera from '../../assets/hints/scaldera.png';
+import moldarach from '../../assets/hints/moldarach.png';
+import koloktos from '../../assets/hints/koloktos.png';
+import tentalus from '../../assets/hints/tentalus.png';
+import g2 from '../../assets/hints/g2.png';
+
+
+
+const pathImages: {[key: string]: string} = {
+    'Ghirahim 1': g1,
+    'Scaldera': scaldera,
+    'Moldarach': moldarach,
+    'Koloktos': koloktos,
+    'Tentalus': tentalus,
+    'Ghirahim 2': g2,
+};
```

Comment:
This is duplicated a bunch of times - it'd be useful to have this in one place.

Reply:
It likely belongs in `itemTracker/Images.ts`, which ultimately is probably in the wrong place now that we're starting to add images in places outside the item tracker.